### DISCUSSION TITLE
Getting this error "numpy.core.multiarray failed to import"

### DISCUSSION THREAD

**Discussion by trmyo** [petercorke/robotics-toolbox-python#331] [General]
Hi, I am getting this error when I tried to import roboticstoolbox. Any solution plz?
Thanks

**Comment by waelbachta** [comment]
> Hi, I am getting this error when I tried to import roboticstoolbox. Any solution plz?
> Thanks

Hello,

You have to update your numpy package : pip3 install numpy -U

**Comment by jhavl** [comment]
Hey @trmyo, the solution from @waelbachta should work fine. Also, in the latest version of the toolbox we have fixed this so it will no longer be an issue.
